workflow/annovar.wdl

```diff
@@ -0,0 +1,148 @@
+version 1.0
+
+# =============================================================== #
+# annovar.wdl                                                     #
+#                                                                 #
+# This workflow annotates a VCF using Annovar.                    #
+#                                                                 #
+# This pipeline has been developed for use by the Kinghorn        #
+# Centre for Clinical Genomics and the Garvan Institute for       # 
+# Medical Research.                                               #
+#                                                                 #
+# Author: Michael Geaghan (micgea)                                #
+# Created: 2023/04/04                                             #
+# =============================================================== #
+
+struct AnnotationRuntime {
+    Int max_retries
+    Int preemptible
+    Int boot_disk_size
+}
+
+workflow Annovar {
+    input {
+        # input vcf
+        File input_vcf
+        # annovar settings
+        Int annovar_mem_mb = 4000
+        Int annovar_disk = 100
+        Int annovar_tmp_disk = 200
+        Int annovar_cpu = 1
+        String annovar_docker = "australia-southeast1-docker.pkg.dev/pb-dev-312200/somvar-images/perl@sha256:1f35086e2ff48dace3b3edeaa2ad1faf1e44c0612e00f00ea0fc1830b576a261"  # :5.34.0
```

Comment:
Just make sure these containers are accessible by the accounts you expect, if you're running on full, the group will be `dataset-full@populationgenomics.org.au` as a shortcut.